tests/unit/wingsail/common/test_lut.py

```diff
@@ -0,0 +1,53 @@
+# import pytest
+
+import numpy as np
+from scipy import interpolate
+
+from controller.common.lut import LUT
+
+
+class TestLUT:
+    # Intialize lookup table
+    look_up_table = [[50000, 5.75], [100000, 6.75], [200000, 7], [500000, 9.75], [1000000, 10]]
+
+    def test_LUT_constructor(self):
+        # set up
+        testLUT = LUT(self.look_up_table)
+
+        # test that LUT return a known value
+        assert testLUT(50000) == 5.75
```

Comment:
In general, use `math.isclose` when comparing floats

Reply:
added `math.isclose` at all assert statements